manifests/p/Paloalto/GlobalProtect/6.1.4/Paloalto.GlobalProtect.installer.yaml

```diff
@@ -0,0 +1,37 @@
+# Created with komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Paloalto.GlobalProtect
+PackageVersion: 6.1.4
+InstallerLocale: en-US
+InstallerType: msi
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2023-06-19
+Installers:
+- Architecture: x86
+  InstallerUrl: https://vpn.extremenetworks.com/global-protect/msi/GlobalProtect.msi

```

Comment:
Since this is a vanity URL, can you please also remove the previous manifest version? You can add the removal just as a commit on this same PR